rustv1/test-utils/src/waiter.rs

```diff
@@ -86,3 +86,31 @@ impl WaiterBuilder {
         Waiter::new(self.max.unwrap_or(MAX_WAIT), self.poll.unwrap_or(POLL_TIME))
     }
 }
+
+#[macro_export]
+macro_rules! wait_on {
+    (
+        $req: expr,
+        $test: expr
+    ) => {
+        wait_on!($crate::waiter::Waiter::default(), $req, $test)
```

Comment:
poll should probably be `poll_interval` to distinguish it from `poll` the Rust async concept

Reply:
Done (`interval`, actually)